src/isIsogram.test.js

```diff
@@ -8,12 +8,28 @@ describe(`Function 'isIsogram':`, () => {
   });
 
   it(`should return a boolean`, () => {
+    const res = isIsogram('word');
 
+    expect(typeof res).toBe('boolean');
   });
 
-  it(`should be case sensitive`, () => {
+  it(`should be not case sensitive`, () => {
```

Comment:
Should be an example with the same letters in different cases and the result should be false. For example: Aa